.github/workflows/test.yml

```diff
@@ -0,0 +1,453 @@
+# This file was auto-generated by .evergreen/make-github-actions
+# at 2021-08-21T15:14:55Z
+---
+name: Run Mongoid Tests
+'on':
+- push
+- pull_request
+jobs:
+  build:
+    name: "${{matrix.ruby}} driver-${{matrix.driver}} mongodb-${{matrix.mongodb}}
+      ${{matrix.topology}}"
+    env:
+      CI: true
```

Comment:
Thank you for adding this, I wanted to check if GH actions sets this automatically but haven't gotten to it. This ensures we definitely expose this as a CI job. 👍